src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Is this really needed? I don't think the code is mocking any final/static methods now?

Reply:
It is needed. The method AbstractItem.getFullName() is final